tap_dbt/streams.py

```diff
@@ -69,11 +73,15 @@ def get_url_params(
         """Return offset as the next page token."""
         params = {}
         _ = context
+        params["limit"] = self.config.get("page_size", 100)
 
         # Next page token is an offset
         if next_page_token:
             params["offset"] = next_page_token
 
+        self.logger.debug("context=%s", context)
+        self.logger.debug("params=%s", params)
+
```

Comment:
Thanks for the review @edgarrmondragon - we are operating an environment with nearly 300,000 job runs that we would like to extract. Having done some performance testing on our implementation, I found a `page size` === `limit=` of 100 was very slow, requiring ~3000 individual API calls to complete. An optimal number was in the many thousands.

The alternative of course is to use incremental loading for the `runs` endpoint, however this becomes fiendishly complex because the only way to limit results from the API is to use the pagination parameters, limit and offset and the `replication_key` we'd use is run ID, which is unrelated to the limit and offset values.